js/layout-javascript/news-feed-code.js

```diff
@@ -157,6 +157,41 @@ DynamicList.prototype.attachObservers = function() {
         console.error(error);
       });
     })
+    .on('click', '.sort-group .list-sort li', function (e) {
+      e.stopPropagation();
+
+      var $sortListItem = $(e.currentTarget);
+      var oldSortOrder = $sortListItem.data('sortOrder');
+      var $sortOrderIcon = $sortListItem.find('i');
+      var sortOrder;
+      var sortField = $sortListItem.data('sortField');;
+      var removeClasses = {
+        'no': 'fa-sort',
+        'asc': 'fa-sort-asc',
+        'desc': 'fa-sort-desc'
+      };
+      var addClasses = {
+        'no': 'fa-sort-asc',
+        'asc': 'fa-sort-desc',
+        'desc': 'fa-sort-asc'
+      }
+      var newSortOrder = {
+        'no': 'asc',
+        'asc': 'desc',
+        'desc': 'asc'
+      };
+
+      sortOrder = newSortOrder[oldSortOrder];
+      _this.resetSortIcons();
+
+      $sortOrderIcon.removeClass(removeClasses[oldSortOrder]).addClass(addClasses[oldSortOrder]);
```

Comment:
You can use one `sortClasses` object instead a `removeClasses` and `addClasses`.

```js
sortClasses = {
  'none': 'fa-sort-asc',
  'asc': 'fa-sort-desc',
  'desc': 'fa-sort-asc'
}

$sortOrderIcon.removeClass(_.values(sortClasses)).addClass(sortClasses[oldSortOrder]);
```